public/member_ops.js

```diff
@@ -152,8 +158,42 @@ function handle_members(event) {
             });
         }
       });
+
+    // Delete Member
     } else if (event.target === deleteMember) {
-    // Your code goes here!
+      // Your code goes here!
+      let headers = { "Content-Type": "application/json" };
+      let csrf_cookie = getCookie("CSRF-TOKEN");
+      if (csrf_cookie) {
+        headers["X-CSRF-Token"] = csrf_cookie;
+      }
+      fetch(`${members_path}/${memberID.value}`, {
+        method: "DELETE",
+        headers: headers,
+      }).then((response) => {
+        if (response.status === 201) {
+          resultsDiv.innerHTML = "";
+          let parag = document.createElement("P");
+          parag.textContent = JSON.stringify(data);
+          resultsDiv.appendChild(parag);
+      } else {
```

Comment:
I had to make a small adjustment to be able to delete a member -- it looks like the member Id used in the path (line 170) is not grabbing the value from the corresponding member Id input field. It should be `memberId2` instead of `memberID`. It works with that change though!

I did notice, though, that when the delete request is successful, the status of the response is `200`, rather than `201`. So your code at lines 175-178 are never run, and instead, the user sees the success message in the alert, because we hit lines 180-188 in that case.